From 10f594cb790178ad54858433a7c8eee304ce6165 Mon Sep 17 00:00:00 2001
From: padarshr <padarshr@codeaurora.org>
Date: Mon, 30 Jul 2018 19:58:38 +0530
Subject: [PATCH 01/12] Remove vendor se-labelling of AOSP's qemu.hw.mainkeys
 prop.

This removes the se-labelling of qemu.hw.mainkeys property
from vendor side. AOSP defined property is no longer
allowed to be se-labelled from vendor side.

Change-Id: Ifeec94911bf61c3fc8b2de3b86349cc88c033071
---
 common/init_shell.te     | 1 -
 common/property.te       | 1 -
 common/property_contexts | 1 -
 common/system_server.te  | 1 -
 4 files changed, 4 deletions(-)

diff --git a/common/init_shell.te b/common/init_shell.te
index c2e6cdd..efc6fbe 100644
--- a/common/init_shell.te
+++ b/common/init_shell.te
@@ -91,7 +91,6 @@ set_prop(qti_init_shell, opengles_prop)
 set_prop(qti_init_shell, mdm_helper_prop)
 set_prop(qti_init_shell, fm_prop)
 set_prop(qti_init_shell, usf_prop)
-set_prop(qti_init_shell, qemu_hw_mainkeys_prop)
 set_prop(qti_init_shell, alarm_boot_prop)
 set_prop(qti_init_shell, boot_animation_prop)
 set_prop(qti_init_shell, debug_gralloc_prop)
diff --git a/common/property.te b/common/property.te
index 897b87e..1579ccc 100644
--- a/common/property.te
+++ b/common/property.te
@@ -71,7 +71,6 @@ type location_prop, property_type;
 
 #properites for init.qcom.sh script
 type rmnet_mux_prop, property_type;
-type qemu_hw_mainkeys_prop, property_type;
 type sys_usb_controller_prop, property_type;
 type sys_usb_configfs_prop, property_type;
 
diff --git a/common/property_contexts b/common/property_contexts
index 073395e..acd71c4 100644
--- a/common/property_contexts
+++ b/common/property_contexts
@@ -72,7 +72,6 @@ persist.rmnet.mux          u:object_r:rmnet_mux_prop:s0
 sys.usb.controller         u:object_r:sys_usb_controller_prop:s0
 sys.usb.configfs           u:object_r:sys_usb_configfs_prop:s0
 sys.usb.tethering          u:object_r:sys_usb_tethering_prop:s0
-qemu.hw.mainkeys           u:object_r:qemu_hw_mainkeys_prop:s0
 ro.dbg.coresight.cfg_file     u:object_r:coresight_prop:s0
 ctl.hbtp                   u:object_r:ctl_hbtp_prop:s0
 vendor.audio.sys.init             u:object_r:audio_prop:s0
diff --git a/common/system_server.te b/common/system_server.te
index c64455c..905f8c0 100644
--- a/common/system_server.te
+++ b/common/system_server.te
@@ -165,7 +165,6 @@ allow system_server sysfs_graphics:file rw_file_perms;
 allow system_server { location_app system_app } :file write;
 allow system_server opengles_prop:file r_file_perms;
 allow system_server graphics_vulkan_prop:file r_file_perms;
-allow system_server qemu_hw_mainkeys_prop:file r_file_perms;
 
 allow system_server hwui_prop:file r_file_perms;
 allow system_server bservice_prop:file r_file_perms;
-- 
2.17.1

